input/resources/au-core-requester.xml

```diff
@@ -2756,6 +2780,8 @@
   <div>
     <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
     <p>The responder <strong>SHALL</strong> support both.</p>
+    <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```

Comment:
Should be SHALL for both responder & requester.

Reply:
fixed